### PR TITLE
net-im/skypeforlinux: bump to 8.20.0.9, delete old

### PR DESCRIPTION
**Pull request by realnc**

### CODE REVIEW

File: net-im/skypeforlinux/skypeforlinux-8.20.0.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+SLOT="0"
+KEYWORDS="~amd64 -*"
+IUSE="pax_kernel"
+
+S="${WORKDIR}"
+QA_PREBUILT="*"
+RESTRICT="mirror bindist strip" #299368
+
+RDEPEND="
+	app-crypt/libsecret
```
[the remark this answers is not in the document]
Comment (realnc) [reply net-im/skypeforlinux/skypeforlinux-8.20.0.9.ebuild]: I don't know what those reasons are though. There's no comment about this in the ebuild. When not removing the flag, repoman would complain.

[the remark this answers is not in the document]
Comment (mgorny) [reply net-im/skypeforlinux/skypeforlinux-8.20.0.9.ebuild]: Yes, this warning indicates a real problem for x32. Unlike your solution which sweeps it under the carpet and pretends that the package works.

[the remark this answers is not in the document]
Comment (realnc) [reply net-im/skypeforlinux/skypeforlinux-8.20.0.9.ebuild]: I see. Wasn't aware of the x32 thing. I added the flag back with a comment mentioning the relevant bug.

File: net-im/skypeforlinux/skypeforlinux-8.20.0.9.ebuild [hunk 2]
```diff
@@ -21,7 +21,7 @@ QA_PREBUILT="*"
 RESTRICT="mirror bindist strip" #299368
 
 RDEPEND="
-	app-crypt/libsecret[${MULTILIB_USEDEP}]
+	app-crypt/libsecret[${MULTILIB_USEDEP}] # Bug 642054
```
Comment (mgorny) [inline net-im/skypeforlinux/skypeforlinux-8.20.0.9.ebuild]: Please test your ebuilds.

Comment (realnc) [reply net-im/skypeforlinux/skypeforlinux-8.20.0.9.ebuild]: Sorry. Fixed.

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**